src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
+  height: 17.5rem;
+  height:100%;
+  margin: 0.5%;
```

Comment:
there should not be a margin on this, otherwise we can't use it in other contexts

Reply:
deleted the margin: 0.5%